include/RAJA/util/macros.hpp

```diff
@@ -126,7 +126,9 @@ RAJA_HOST_DEVICE RAJA_INLINE void RAJA_UNUSED_VAR(T &&...) noexcept
 RAJA_HOST_DEVICE
 inline void RAJA_ABORT_OR_THROW(const char *str)
 {
-#if defined(__CUDA_ARCH__)
+#if defined(RAJA_ENABLE_TARGET_OPENMP)
+  printf ( "%s\n", str );
```

Comment:
Hi @trws @rhornung67 , I'm not sure how to gracefully exit an omp target region when there might be an ungraceful error. For now, I'm just printing the error, but if you have any suggestions please let me know.

The original problem is that during OpenMP Target linking, NVCC complains that it cannot find some std:: calls, e.g. line 152 below. We are relying on the host compiler definition here, so for OpenMP Target compilation this will be difficult to avoid. This current solution also hurts legitimate OpenMP non-target code compiled under OpenMP Target, because we're no longer calling std::abort even while on the host. Alternatively, I can just macro out the call to RAJA_ABORT_OR_THROW in View.hpp when building for OpenMP Target (maybe this is the better workaround for now).